packages/ListBox/src/components/Popover/Popover.js

```diff
@@ -21,9 +21,9 @@ export default function Popover(props) {
   const { children, zIndex } = props;
 
   return (
-    <PopoverStyled {...props} isOpen={state.isOpen} maxWidth={state.triggerWidth} offset={0} zIndex={zIndex}>
+    <sc.PopoverStyled {...props} isOpen={state.isOpen} maxWidth={state.triggerWidth} offset={0} zIndex={zIndex}>
```

Comment:
`<sc.PopoverStyled>` seems redundant to me. `<sc.Popover>` tells me the same thing with fewer words.